newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -852,7 +890,7 @@ func findDashboardWidgetFilterCurrentDashboard(d *schema.ResourceData) ([]interf
 	var widgetList []interface{}
 
 	pages := d.Get("page").([]interface{})
-	selfLinkingWidgets := []string{"widget_bar", "widget_pie", "widget_table"}
+	selfLinkingWidgets := []string{"widget_bar", "widget_pie", "widget_table", "widget_log_table"}
```

Comment:
wasn't sure if this belongs here or not.

Reply:
I don't think this is needed. `widget_log` doesn't seem to have filter capabilities.